portfolio/src/main/webapp/script.js

```diff
@@ -14,6 +14,13 @@
 
 async function getData() {
     const response = await fetch('/data');
-    const data = await response.text();
-    document.getElementById('data-container').innerHTML = data;
+    const comments = await response.json();
+    let html = "";
+    for(const comment of comments) {
+        html += "<div id=\"comment\">\n";
+        html += "<p>" + comment.message + "</p>\n";
```

Comment:
I don't know if you need the `\n` at the end of the line. If you look at [this fiddle](https://jsfiddle.net/p7jLh3sa/) for example, you can just mash all of the `<p>` tags together and it will line break for you. Of course, if you weren't programmatically generating the HTML that would be horrible to look at.

Reply:
I thought it would make a difference when someone inspects the html through their browser, but I just verified by removing them that the `\n` at the end of each line is indeed unnecessary.